nessie/src/test/java/org/apache/iceberg/nessie/TestNessieView.java

```diff
@@ -0,0 +1,337 @@
+        .withSchema(ALTERED)
+        .withDefaultNamespace(VIEW_IDENTIFIER.namespace())
+        .commit();
+
+    getView(KEY); // sanity, check view exists
+    // check parameters are in expected state
+    String expected = temp.toUri() + DB_NAME + "/" + viewName;
+    Assertions.assertThat(getViewBasePath(viewName)).isEqualTo(expected);
+
+    Assertions.assertThat(metadataVersionFiles(viewLocation)).isNotNull().hasSize(2);
+
+    verifyCommitMetadata();
+  }
+
+  @Test
+  public void testRenameWithTableReference() throws NessieNotFoundException {
+    String renamedViewName = "rename_view_name";
+    TableIdentifier renameViewIdentifier =
+        TableIdentifier.of(VIEW_IDENTIFIER.namespace(), renamedViewName);
+
+    ImmutableTableReference fromTableReference =
+        ImmutableTableReference.builder()
+            .reference(catalog.currentRefName())
+            .name(VIEW_IDENTIFIER.name())
+            .build();
+    ImmutableTableReference toTableReference =
+        ImmutableTableReference.builder()
+            .reference(catalog.currentRefName())
+            .name(renameViewIdentifier.name())
+            .build();
+    TableIdentifier fromIdentifier =
+        TableIdentifier.of(VIEW_IDENTIFIER.namespace(), fromTableReference.toString());
+    TableIdentifier toIdentifier =
+        TableIdentifier.of(VIEW_IDENTIFIER.namespace(), toTableReference.toString());
+
+    catalog.renameView(fromIdentifier, toIdentifier);
+    Assertions.assertThat(catalog.viewExists(fromIdentifier)).isFalse();
+    Assertions.assertThat(catalog.viewExists(toIdentifier)).isTrue();
+
+    Assertions.assertThat(catalog.dropView(toIdentifier)).isTrue();
+
+    verifyCommitMetadata();
+  }
+
+  @Test
+  public void testRenameWithTableReferenceInvalidCase() {
+    String renamedViewName = "rename_view_name";
+    TableIdentifier renameViewIdentifier =
+        TableIdentifier.of(VIEW_IDENTIFIER.namespace(), renamedViewName);
+
+    ImmutableTableReference fromTableReference =
+        ImmutableTableReference.builder()
+            .reference("Something")
+            .name(VIEW_IDENTIFIER.name())
+            .build();
+    ImmutableTableReference toTableReference =
+        ImmutableTableReference.builder()
+            .reference(catalog.currentRefName())
+            .name(renameViewIdentifier.name())
+            .build();
+    TableIdentifier fromIdentifier =
+        TableIdentifier.of(VIEW_IDENTIFIER.namespace(), fromTableReference.toString());
+    TableIdentifier toIdentifier =
+        TableIdentifier.of(VIEW_IDENTIFIER.namespace(), toTableReference.toString());
+
+    Assertions.assertThatThrownBy(() -> catalog.renameView(fromIdentifier, toIdentifier))
+        .isInstanceOf(IllegalArgumentException.class)
+        .hasMessage(
+            "Cannot rename ICEBERG_VIEW 'view' on reference 'Something' to 'rename_view_name' on reference 'iceberg-view-test' : source and target references must be the same.");
```

Comment:
```suggestion
            "Cannot rename ICEBERG_VIEW 'view' on reference 'Something' to 'rename_view_name' on reference 'iceberg-view-test': source and target references must be the same.");
```

Reply:
there is an unnecessary space. The same is for the error message for `ICEBERG_TABLE`